components/tournaments_summary_table/commons/tournaments_summary_table.lua

```diff
@@ -283,15 +283,15 @@ function TournamentsSummaryTable.row(eventInformation, type)
 	local iconFile = ''
 	if String.isNotEmpty(eventInformation.icon) then
 		local iconInput = string.gsub(eventInformation.icon, 'File:', '')
-		if mw.title.new('Media:' .. iconInput).exists then
+		if (mw.title.new('Media:' .. iconInput) or {}).exists then
```

Comment:
Potential improvement that would have made the example actually usable without changes:
```suggestion
		if (mw.title.new('Media:' .. mw.uri.decode(iconInput)) or {}).exists then
```
Not sure whether we want/need this here, or maybe rather in the infobox already?

Reply:
imo in infobox is preferable

Reply:
Then the followup question, is it actually needed?
Same for whether this page.exists check is needed at all.
For the actual file usage, it works with percent encoding as well.

Reply:
Maybe that's more of a question to @FO-nTTaX, does the TournamentsMenu require the iconfile to exist for it to not break?

Reply:
Yeah the existence of the file is checked for in the extension. Also you should not use `Media:`, please use `File:`

Reply:
Would it make sense to add the check for the iconDarkFile as well there?

Reply:
Might be worthwhile, I'm accepting pull requests :P

Reply:
`Template:LeagueIconSmall/mainpageTST` invokes `Module:LeagueIcon` which does empty checks iirc